packages/core/src/services/time-series.ts

```diff
@@ -84,8 +84,10 @@ export function formatTick(
 	i: number,
 	allTicks: Array<number>,
 	interval: string,
-	timeScaleOptions: TimeScaleOptions
+	timeScaleOptions: TimeScaleOptions,
+	options
```

Comment:
```suggestion
	localeOptions
```

Reply:
let's just pass `options.locale` into this function rather than the full options